### PR TITLE
Fix reading of network namespace inodes

### PR DESCRIPTION
**Pull request by tomwilkie**
Use stat instead of lstat.

Need testing on pre-3.8 kernels.

### CODE REVIEW

**Comment by 2opremio** [comment]
Just confirmed that stat works on kernel 3.2 on which `/proc/PID/ns/net` are not symlinks

```
vagrant@precise64:~$ uname -a
Linux precise64 3.2.0-23-generic #36-Ubuntu SMP Tue Apr 10 20:39:51 UTC 2012 x86_64 x86_64 x86_64 GNU/Linux
vagrant@precise64:~$ sudo ls -la /proc/1/ns/net
-r-------- 1 root root 0 Feb  2 09:53 /proc/1/ns/net
vagrant@precise64:~$ sudo stat /proc/1/ns/net --format %i
9607
vagrant@precise64:~$ sudo stat -L /proc/1/ns/net --format %i
9607
```

**Comment by tomwilkie** [comment]
Do you want to stick in some code to do that on pre 3.8 kernels? We can switch based on kernel versions.

**Comment by 2opremio** [comment]
Done

**Comment by tomwilkie** [comment]
I put some minor comments here: https://github.com/weaveworks/scope/commit/5e585a893b8be8315e0118705b4a15847fbe4e8d

I think they got blown away by the push -f

File: probe/endpoint/procspy/proc.go [hunk 1]
```diff
@@ -4,16 +4,22 @@ package procspy
 
 import (
 	"bytes"
+	"log"
 	"path/filepath"
 	"strconv"
 	"syscall"
 
+	"github.com/armon/go-metrics"
+	"github.com/hashicorp/go-version"
```
Comment (tomwilkie) [inline probe/endpoint/procspy/proc.go]: Need a new line after this.

**Comment by tomwilkie** [comment]
LGTM with 1 comment, pending tests.

**Pull request merged**